src/main/kotlin/no/nav/familie/ks/barnehagelister/domene/BarnehagelisteService.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ks.barnehagelister.domene
+
+import no.nav.familie.ks.barnehagelister.repository.BarnehagelisterRepository
+import no.nav.familie.ks.barnehagelister.rest.dto.BarnehagelisteStatus
+import no.nav.familie.ks.barnehagelister.task.MottattBarnehagelisteTask
+import no.nav.familie.prosessering.internal.TaskService
+import org.slf4j.LoggerFactory
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class BarnehagelisteService(
+    private val barnehagelisterRepository: BarnehagelisterRepository,
+    val taskService: TaskService,
```

Comment:
Denne kan kanskje være `private`? 

```suggestion
    private val taskService: TaskService,
```

Reply:
Yess, godt poeng! :D